src/windows/ai_media_panel.py

```
@@ -190,6 +194,35 @@ def filter_tags(self, text):
+            if not f or not isinstance(getattr(f, "data", None), dict):
+                return False
+            if f.data.get("media_type") != "video":
+                return False
+            files_model = getattr(get_app().window, "files_model", None)
+            if not files_model or not hasattr(files_model, "_tag_file_async"):
+                return False
+            self._auto_retag_requested.add(file_id)
+            log.info("AIMediaPanel: auto re-tagging %s (no usable metadata)", file_id)
+            files_model._tag_file_async(file_id)
+            return True
+        except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)